### PR TITLE
[cases] AllCases hi-fis

### PR DESCRIPTION
**Pull request by ronniebeggs**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Refactored `AllCases` screen to match Figma Hi-Fis from final hackathon.
- Replace react-native images with expo-images.
- Standardized styling across related components/screens.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- Ensure that styling matches Figma hi-fis.
- Check that styling logic is consistent and correct across all related components/screens.

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

- Re-integrate the QR code camera button with Hi-Fi designs.
- Create standardized global font styles.
- Create a project style guide that establishes design standards/best practices.
- Add alternative text for images.

### Online sources
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

[Shadows Documentation](https://reactnative.dev/docs/shadow-props)
[Expo Image Documentation](https://docs.expo.dev/versions/latest/sdk/image/)

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[CaseScreen Hi-Fis](https://github.com/calblueprint/impact-fund/pull/54)
[Case Queries](https://github.com/calblueprint/impact-fund/pull/12)

CC: @ronniebeggs

### CODE REVIEW

File: src/Components/CaseCard/CaseCard.tsx [hunk 1]
```diff
@@ -15,20 +22,33 @@ function CaseCard(caseData: Case) {
...
+            {caseData.title}
+          </Text>
+
+          <View style={[styles.statusContainer, statusColor.background]}>
+            <Text style={[styles.statusText, statusColor.text]}>
+              {caseData.caseStatus}
+            </Text>
+          </View>
+        </View>
+        <Image
```
Comment (stephaniewong2) [inline src/Components/CaseCard/CaseCard.tsx:37]: Should we add a blur hash for images? Not sure what looks better.

File: src/Components/CaseSummaryCard/CaseSummaryCard.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { View, Text, Image, TouchableOpacity } from 'react-native';
```
Comment (stephaniewong2) [inline src/Components/CaseSummaryCard/CaseSummaryCard.tsx]: We should switch this to expo `Image` as well

File: src/app/(BottomTabNavigation)/AllCases/CaseSummaryScreen/[caseUid].tsx [hunk 3]
```diff
@@ -28,30 +28,38 @@ export default function CaseSummaryScreen() {
       {caseData === undefined ? (
         <Text>Loading...</Text>
       ) : (
-        <ScrollView
-          style={styles.scrollContainer}
-          showsVerticalScrollIndicator={false}
-        >
-          <Image
```
Comment (stephaniewong2) [inline src/app/(BottomTabNavigation)/AllCases/CaseSummaryScreen/[caseUid].tsx:35]: expo `Image` here as well

**Review by stephaniewong2 — COMMENTED** [review]
Looks good to me! I wasn't able to review styling thoroughly because I can't seem to load it on Expo. Besides that, we should make sure all react native `Image`s are replaced with expo.

File: src/Components/CaseCard/CaseCard.tsx (hunk 1 above)
Comment (ronniebeggs) [reply src/Components/CaseCard/CaseCard.tsx:37]: The default blur hash on the expo docs doesn't look that great IMO. I'll experiment with it tho.

**Pull request merged**